### ISSUE TITLE
Dependent start & end dates not working

### ISSUE THREAD

**Issue by kjlloydie** [laravel-filament/filament#2224]
(As mentioned in discussion #2201)

### Package

filament/filament

### Package Version

v2.0

### Laravel Version

v9.0

### Livewire Version

v2.5

### PHP Version

PHP 8.1

### Bug description

I have a form with start and end dates and I'd like to ensure the end date isn't before the start date. I've set it up roughly like this:
```php
DateTimePicker::make('start_date')
    ->withoutTime()
    ->reactive()
    ->displayFormat('d/m/Y'),
DateTimePicker::make('end_date')
    ->withoutTime()
    ->reactive()
    ->displayFormat('d/m/Y')
    ->minDate(function (Closure $get) {
        return $get('start_date');
    }),
```
which seems to work the first time the form is created, but when I change _start_date_ the calendar/selector on _end_date_ disappears, like this:
<img width="355" alt="Screenshot 2022-04-22 at 2 56 50 PM" src="https://user-images.githubusercontent.com/489813/164801162-ef1fea10-385a-41c5-8916-8245cd80cdba.png">

### Steps to reproduce

1. Setup two DateTimePicker fields
2. Set the `minDate` on one field to use the value of the other
3. On the form change the first date field value

**Comment by saade** [comment]
any console errors?

**Comment by kjlloydie** [comment]
No, afraid not.

**Comment by danharrin** [comment]
Fixed by #2314, you don't need the second date to be `reactive()`, only fields that are *depended upon* should be reactive, not those that are *dependant*.
